frontend/public/components/factory/modal.tsx

```diff
@@ -65,11 +66,21 @@ export const createModalLauncher: CreateModalLauncher = (Component) => (props) =
 export const ModalTitle: React.SFC<ModalTitleProps> = ({
   children,
   className = 'modal-header',
+  enableCloseButton,
+  close,
 }) => (
   <div className={className}>
-    <h1 className="pf-c-title pf-m-2xl" data-test-id="modal-title">
+    <div className="pf-c-title pf-m-2xl" data-test-id="modal-title">
```

Comment:
No need to change it from `h1` to `div`.

Reply:
done